package.json

```diff
@@ -13,11 +13,13 @@
   "scripts": {
     "build": "lerna run --parallel build",
     "dev": "lerna run --parallel dev",
-    "lint:fix": "lerna run lint:fix",
+    "test:components": "lerna exec --scope @carto/airship-components -- npm run test",
```

Comment:
👍I guess `exec` allows scope, and that's why we don't use `run` here...

Reply:
Yes, it does for sure. But if we use run, it doesn't show the colored output, which is so useful. 😞